src/client/assets/icons/etherscan.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.83029 12.2221C4.83029 11.6269 5.28313 11.1805 5.88691 11.1805H7.54734C8.15112 11.1805 8.60396 11.6269 8.60396 12.2221V18.6207C8.75491 18.6207 9.0568 18.4719 9.35869 18.4719C9.81153 18.3231 9.96249 18.0255 9.96249 17.5791V9.69244C9.96249 9.09723 10.4153 8.65082 11.0191 8.65082H12.6795C13.2833 8.65082 13.7362 9.09723 13.7362 9.69244V16.9839C13.7362 16.9839 14.189 16.8351 14.6418 16.6862C14.9437 16.5374 15.0947 16.2398 15.0947 15.9422V7.16277C15.0947 6.56756 15.5475 6.12115 16.1513 6.12115H17.8117C18.4155 6.12115 18.8683 6.56756 18.8683 7.16277V14.3054C20.3778 13.2638 21.8873 11.9245 23.0949 10.4365C23.3968 9.99005 23.5477 9.39485 23.3967 8.79963C21.7363 4.03789 17.2079 0.912982 12.2267 0.764178C5.58504 0.615374 0 6.12114 0 12.6685C0 14.7518 0.603796 16.8351 1.66042 18.6207C1.96232 19.0671 2.5661 19.3647 3.16989 19.3647C3.47178 19.3647 3.92461 19.3647 4.37745 19.2159C4.83029 19.2159 5.13218 18.7695 5.13218 18.3231L4.83029 12.2221Z" fill="black"/>
+<path d="M4.82976 22.4897C10.2638 26.3586 17.8112 25.1682 21.7358 19.8112C23.2453 17.7279 24 15.3471 24 12.8174C24 12.5198 24 12.2222 24 12.0734C19.4716 18.472 11.3205 21.5969 4.82976 22.4897Z" fill="black"/>
```

Comment:
same "remove fill="black" so we can change icon color in different themes with css"